x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -42,6 +44,88 @@ const ELASTICSEARCH_PASSWORD = 'ELASTICSEARCH_PASSWORD';
  */
 const LOGIN_API_ENDPOINT = '/internal/security/login';
 
+// these roles are derived from https://docs.google.com/spreadsheets/d/1SGjXVS3gQyRxCN1MF2Pq3pgos3vtfpROqTvi4z-0WiU/edit#gid=0
```

Comment:
Since this isn't a public worksheet perhaps we just reference this internally since others won't have access to it?

Reply:
@MadameSheema suggested putting a link to the spreadsheet here to make it obvious where these roles originated. I think she had a point in making sure we documented the source of these but maybe a comment in the code pointing to an internal spreadsheet is not the best place. Is there somewhere else we could do this? Maybe just reference it directly in the description of the PR?